spec/helpers/observableMatcher.ts

```diff
@@ -1,4 +1,5 @@
 import * as _ from 'lodash';
+import * as chai from 'chai';
```

Comment:
I'm not sure why this fixes the issue (or rather how other tests are working without this), but without this, all tests for the concat operator fail with `chai is not defined` error.

If there is a better solution, please let me know!

Reply:
I was looking through some not-yet-merged PRs - 'cause other contributors have already converted some tests to use `run` and I wanted to make sure your efforts weren't going to be wasted, with the same tests converted twice - and I found a PR that makes this change: https://github.com/ReactiveX/rxjs/pull/5037 I've merged that, so you can rebase this PR.